pkg/preparer/orchestrate_test.go

```diff
@@ -113,25 +135,10 @@ func TestPreparerWillNotLaunchIfSHAIsTheSame(t *testing.T) {
 		currentManifest: testManifest,
 	}
 
-	var p *Preparer
+	p := testPreparer(testManifest)
 	success := p.installAndLaunchPod(testManifest, testPod, logging.DefaultLogger)
 
 	Assert(t).IsTrue(success, "Should have been a success to prevent retries")
 	Assert(t).IsFalse(testPod.launched, "Should not have attempted to launch")
 	Assert(t).IsTrue(testPod.installed, "Should have installed")
 }
-
-func TestInstallReturnsFalseIfManifestErrsOnRead(t *testing.T) {
```

Comment:
Agreed, that this is a good test to keep - this ensures that the reality store being corrupt for whatever reason doesn't stop a scheduled install.